backend/src/api/cycle.js

```diff
@@ -32,6 +36,37 @@ router.get(
   })
 )
 
+// get all cycles belonging to a workspace (either current, outdated, or both)
+// TODO: when authentication server is integrated, only show the cycles
+// to the workspace's owner. Otherwise, 404.
```

Comment:
if they request a workspace that doesn't belong to them, should it be a 404 or just an empty response?

Reply:
hmmmm empty response suggests workspace exists but cycles not created, so I think 404 is better

Reply:
that's true, what about 401? saying that it's not found is leaving a bad taste in my mouth somehow

Reply:
I think 403 is actually the one we should use lol

https://stackoverflow.com/questions/50143518/401-unauthorized-vs-403-forbidden-which-is-the-right-status-code-for-when-the-u?rq=1

Reply:
lmao there we have it